workitem/workitem_repository.go

```diff
@@ -17,10 +17,15 @@ import (
 	uuid "github.com/satori/go.uuid"
 )
 
+const orderValue = 1000
+const above = "above"
+const below = "below"
+
 // WorkItemRepository encapsulates storage & retrieval of work items
 type WorkItemRepository interface {
 	Load(ctx context.Context, ID string) (*app.WorkItem, error)
 	Save(ctx context.Context, wi app.WorkItem, modifierID uuid.UUID) (*app.WorkItem, error)
+	Reorder(ctx context.Context, direction string, targetID string, wi app.WorkItem, modifierID uuid.UUID) (*app.WorkItem, error)
```

Comment:
@aslakknutsen Would have been nice to have this for the work item ID as well ;) . That is also just a arbitrary string that needs to be convertible into a number. But that's a different story ;)